Makefile

```
@@ -0,0 +1,79 @@
+# Makefile for building Docker Compose from backends/advanced directory
+
+.PHONY: all help build-backend build-no-cache up-backend down-backend logs clean build-langfuse build-asr-services up-langfuse down-langfuse up-asr-services down-asr-services
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)